doc/tech/usecases.md

```diff
@@ -0,0 +1,194 @@
+
+Edge is considered as one of the main enablers to 5G use cases adoption. There are a number of 5G challenges that Edge will support solve so as to implement the 5G requirements defined by "ITU-2020 for 5G." Edge is required to support enhanced mobile broadband (eMBB), massive machine-type communications (mMTCs) and ultra-reliable low-latency communications (uRLLCs).
+
+With respect to monetization, Edge will help DSPs with their Return on investment (ROI) on 5G and cloud as a number of new services require the Edge. For example, services such as Cloud Gaming, Assisted Reality/Virtual Reality (AR/VR), 5G private networks, and vRAN.
+
+<a name="5.1.2"></a>
+### 5.1.2 Objective
+
+Based on CNTT’s goal and purpose is to develop a robust cloud infrastructure model and define a limited set of discrete architectures built on that model that can be tested and validated for use across the entire member community.
+
+Extend CNTT's Scope beyond the Regional and National Data Center cloud infrastructures to the Edge. The Edge is a disruptive use case where CNTT can add value especially as there are a number of scattered initiatives under various Standard Development Organisations (SDO) and open source communities."
+
+Edge Stream related activities :
+
+To Harmonize the work under Standard Development bodies or Open source communities
+To build Common Cloud infrastructure based on CNTT principles that can be consum ed by any operator
+To build Cloud infrastructure that can scale over hundreds of thousands of nodesand cover the Edge Telco use cases that can help operators to monetize the NFV/SDN journey
+Modify the existing RM and RA-s so that they are aligned with the edge requirements
+
+<a name="5.1.3"></a>
+### 5.1.3 Approach & Scope
+
+All gaps under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s will be identified and Edge requirements will be fulfilled  under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s  
+
+Edge scope under CNTT will cover :
+
+-	Define Edge locations based on use case 
+-	Define some guideline about external factors that can affect the edge like WAN latency based on telco use case 
+-	Define VNF/CNF profiles (if needed) specific to Edge use cases such as profiles required for acceleration technologies 
+-	Define resource requirements in terms of compute, storage and networking; new concepts, such as hyper converged infrastructure, can     be introduced
+-	Define different Architecture models as no one size fits all"
+-	
+
+Out of Scope 
+
+-	The API that can be exposed to 3rd party application developer 
+-	The Architecture of VNF/CNF ( CNTT Guideline only )
```

Comment:
what does it mean by VNF/CNF architecture? Is this about the application architecture?

Reply:
Yes , @pgoyal01

Reply:
i will rephrase it "VNF/CNF architecture"